qfieldcloud_sdk/sdk.py

```diff
@@ -617,6 +617,33 @@ def job_status(self, job_id: str) -> Dict[str, Any]:
 
         return resp.json()
 
+    def push_delta(self, project_id: str, delta_filename: str) -> Dict[str, Any]:
+        """Push a delta file to a project.
+
+        Args:
+            project_id: Project ID.
+            delta_filename: Path to the delta JSON file.
+
+        Returns:
+            A dictionary containing the response from the server.
+        """
+        try:
+            with open(delta_filename, "r") as delta_file:
+                files = {"file": delta_file}
+                resp = self._request(
+                    "POST",
+                    f"deltas/{project_id}/",
+                    files=files,
```

Comment:
```suggestion
                    files={
                      "file": delta_file
                    },
```
I would prefer to reduce the number of parasite variables.